Assets/_BForBoss/_Utility/Editor/PropertyDrawers/ResolverPropertyDrawer.cs

```diff
@@ -0,0 +1,369 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using PerigonGames;
+using UnityEditor;
+using UnityEditor.IMGUI.Controls;
+using UnityEngine;
+using UnityEngine.Rendering.VirtualTexturing;
+using Object = System.Object;
+
+namespace BForBoss
+{
+    [CustomPropertyDrawer(typeof(ResolveAttribute))]
+    public class ResolverPropertyDrawer : PropertyDrawer
+    {
+        private const float RESOLVER_WIDTH = 80f;
+        private const float OBJECT_FIELD_BUTTON_WIDTH = 18f;
+        private const float RESOLVER_BUTTON_WIDTH = 20f;
+        private const float ENUMERABLE_BOX_OFFSET = 25f;
+        private const float STRUCT_OFFSET = 15f;
+
+        private bool _isContentInitialized = false;
+        private bool _includeInactiveGameObjects;
+        
+        private bool _isList = false;
+        private bool _isArray = false;
+        private bool _isDeclaredInStruct = false;
+        
+        //Button Content
+        private GUIContent _childResolveContent;
+        // private GUIContent _sceneResolveContent;
```

Comment:
Are all these needed or...

Reply:
Probably not, was going to make a button for Sibling gameObjects but not sure if the use case is used enough to justify it